conf/messages

```diff
@@ -819,7 +819,7 @@ ated.property-details-period-error.general.taxAvoidancePromoterReference = There
 ated.property-details-period.supportingInfo.title = Do you have any supporting information to add? (optional)
 ated.property-details-period.supportingInfo.header = Do you have any supporting information to add? (optional)
 ated.property-details-period.supportingInfo = Supporting information (optional)
-ated.property-details-period.supportingInfo.hint = See the <a href="https://www.gov.uk/government/publications/stld-annual-tax-on-enveloped-dwellings-ated/annual-tax-on-enveloped-dwellings-returns-guidance#sec5" target="_blank">guidance for more information</a>.
+ated.property-details-period.supportingInfo.hint = <a href="https://www.gov.uk/government/publications/stld-annual-tax-on-enveloped-dwellings-ated/annual-tax-on-enveloped-dwellings-returns-guidance#sec5" target="_blank">How to send additional information about your return (opens in new tab)</a>.
```

Comment:
So not sure it is correct to have the fullstop outside of the <a> as when you use VoiceOverf it will read the link out and then separately reads the fullstop out after ?

Reply:
Screen readers do not do that. VO would read "link... How to send additional information about your return (opens in new tab)" then nothing. It would read out "full stop" or "period", even if left to read the page verbosely, which isn't something done that often. Screenreader users would generally navigate pages like this using shortcuts directly to the links using a menu system or tabbing or using the tool to highlight things like headings etc